src/panels/lovelace/editor/view-editor/hui-edit-view.ts

```diff
@@ -87,6 +87,17 @@ export class HuiEditView extends LitElement {
     return this.shadowRoot!.querySelector("ha-paper-dialog")!;
   }
 
+  private get _viewConfigTitle(): string {
+    const title = this.hass!.localize(
+      "ui.panel.lovelace.editor.edit_view.header"
+    );
+    if (!this._config || !this._config.title || this._config.title === "") {
+      return title;
+    }
+
+    return `${this._config.title} ${title}`;
```

Comment:
We should provide the title as a variable to the translation. So the translator can move the order of the words when needed.

Reply:
@bramkragten If we make the title as a translation variable though, then we wouldn't be able to data bind to the title field, and include the 'view configuration'

Reply:
Not sure I understand what you are saying I suggest:
```js
this._config.title ? 
    this.hass!.localize("ui.panel.lovelace.editor.edit_view.header_name", name, ${this._config.title}) : 
    this.hass!.localize("ui.panel.lovelace.editor.edit_view.header")
```

Reply:
I'm not sure I am following either. Because config.title is a dynamic value, it doesn't seem like we can move that to the translation variables like you mentioned, so how do you suggest that we allow the translator to change the order of the translation for that value?

Oh wait a second. So you are suggesting passing all of that in to the localize function, and having the translator take care of it? 

Where is the code for the localize function to change the parameters for the function? Perhaps I am confused about how the hass.localize function works.

Reply:
https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/common/translations/localize.ts#L75-L79

Reply:
We can pass arguments to the localize function, and add placeholders (`{placeholder}`) to the translations text. These placeholders are replaced by the values we pass in as arguments.